core/arch/arm/include/kernel/early_ta.h

```diff
@@ -18,5 +19,11 @@ struct early_ta {
 
 #define __early_ta __section(".rodata.early_ta" __SECTION_FLAGS_RODATA)
 
+#define for_each_early_ta(_ta) \
```

Comment:
Please remove the original implementation in `core/arch/arm/kernel/early_ta.c`

Reply:
that is done. this was in orignal patch then after multiple rebases I lost that somehow.